### DISCUSSION TITLE
UpdateAsync, Set and null

### DISCUSSION THREAD

**Discussion by mxmissile** [linq2db/linq2db#4208] [Q&A] [answered]
Getting "The call is ambiguous between the following methods..." when setting a value to null using `Set()`:

```
    await _db.Orders
            .Where(x => x.Id == id)
            .Set(x => x.Signed, null) // error is here
            .UpdateAsync();
```
Signed is `DateTime?` 
Nullable has been enabled:

```
   <TargetFramework>net7.0</TargetFramework>
   <Nullable>enable</Nullable>
```

How would I set the the value to be null here?

**Comment by viceroypenguin** [comment, marked as answer]
Try `Set(x => x.Signed, x => null)`

**Reply by MaceWindu** [reply]
or `Set(x => x.Signed, (DateTime?)null)`
